console/console-init/ui/src/modules/address-space/components/AddressSpaceReview/AddressSpaceReview.tsx

```diff
@@ -160,13 +261,12 @@ export const AddressSpaceReview: React.FC<IAddressSpaceReviewProps> = ({
           <AceEditor
             mode="xml"
             theme="github"
+            width="auto"
             fontSize={14}
-            onChange={() => {}}
             value={data.addressSpaceCommand}
             name="UNIQUE_ID_OF_DIV"
             editorProps={{ $blockScrolling: true }}
             style={{
-              width: 700,
               border: "1px solid",
```

Comment:
Can you try to use css class, AceEditor support className as well and use patterfly global variables for color.